src/AvaloniaEdit/Editing/TextArea.cs

```diff
@@ -115,7 +115,7 @@ protected override void OnTemplateApplied(TemplateAppliedEventArgs e)
             if (e.NameScope.Find("PART_CP") is ContentPresenter contentPresenter)
             {
                 contentPresenter.Content = TextView;
-                ((ISetLogicalParent)TextView).SetParent(this);
+                //((ISetLogicalParent)TextView).SetParent(this); Crash on newest avalonia build
```

Comment:
I think it was needed for bindings to work. Check that it didn't break anything. Can you provide he stack trace?

Reply:
My guess was that the new update sets the parent automaticly. I tested a lot and had no issues with just uncommenting it. The exception was that Parent is already set. Can send a full stack trace later

Reply:
@danwalmsley did the same in #89

Reply:
Then please remove this line